betty/tests/test_cache.py

```diff
@@ -0,0 +1,144 @@
+from pathlib import Path
+from typing import Sequence
+
+import pytest
+from pytest_mock import MockerFixture
+
+from betty.cache import MemoryCache, FileCache
+from betty.locale import DEFAULT_LOCALIZER
+
+
+class TestFileCache:
+    async def test_scopes(self, tmp_path: Path) -> None:
```

Comment:
Extract the test methods that cover `Cache` implementations into a test base class.